### DISCUSSION TITLE
Feature Request: Remove filter icon from Header, but keep it in Toolbar

### DISCUSSION THREAD

**Discussion by ryanmk54** [KevinVandy/material-react-table#7] [Ideas]
In the image below, WHS is only 3 characters. If the filter icon were removed, it would give more room to the other columns. Could there be an option to remove the filter icon from the header, but keep it in the Toolbar?
![image](https://user-images.githubusercontent.com/1811468/175162841-4773acc3-b458-4a32-a840-3494bdf05753.png)

**Comment by KevinVandy** [comment]
I've been meaning to address the extra space it takes up for a bit. I might redesign the filter and sort with a more `<sup>` position so they take up less space, and they could even be position absolute in order to not take up that space. I'll see

**Comment by ryanmk54** [comment]
Improved in commit 9507b64445c04cec0d148988a145b1911aa0c18a
Thank you!

**Comment by KevinVandy** [comment]
You know, I'm probably going to end up hiding this filter icon unless a filter is applied so that its not so confusing what clicking it does. That's how it was originally anyway

**Reply by AlexChadwickP** [reply]
Would it be worth considering opening a menu on right-click rather than having an icon?
